pages/linux/qrcp.md

```diff
@@ -0,0 +1,28 @@
+# qrcp
+
+> A file transfer tool.
+> More information: <https://github.com/claudiodangelis/qrcp>.
+
+- Send a file or directories:
+
+`qrcp send {{path/to/file_or_directory path/to/file_directory ...}}`
+
+- Receive files:
+
+`qrcp receive`
+
+- [z]ip content before transferring:
```

Comment:
```suggestion
- Compress content before transferring:
```

Reply:
It's using the zip archiving format, so wouldn't it be better to write zip?